spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerAutoConfiguration.java

```diff
@@ -35,13 +35,15 @@
 @AutoConfigureOrder(Ordered.HIGHEST_PRECEDENCE)
 public class LocalDeployerAutoConfiguration {
 
-	@Bean
+	public static final String DEPLOYER = "Local";
+
+	@Bean(DEPLOYER + "AppDeployer")
 	@ConditionalOnMissingBean(AppDeployer.class)
 	public AppDeployer appDeployer(LocalDeployerProperties properties) {
```

Comment:
maybe we should just handle this by naming the methods `localAppDeployer` and `localTaskLauncher`